glambie/data/timeseries.py

```diff
@@ -78,6 +79,46 @@ def as_dataframe(self):
                              'glacier_area_observed': self.glacier_area_observed
                              })
 
+    def as_cumulative_timeseries(self) -> pd.DataFrame:
+        """
+        Calculates the cumulative timeseries
+
+        Returns
+        -------
+        pd.DataFrame
+            Dataframe containing the cumulative changes
+            Instead of start_dates and end_dates, the column 'dates' is used
+        """
+        if not self.is_cumulative_valid():
+            warnings.warn("WARNING: Cumulative timeseries may be invalid. This may be due to the timeseries containing \
```

Comment:
errant backslash here, and there's no need to repeat the word "warning" after python's own "warning". I think you mean:

```
 warnings.warn("Cumulative timeseries may be invalid. This may be due to the timeseries containing"
"gaps or overlapping periods.")
```